x-pack/plugins/observability/public/components/shared/exploratory_view/header/metric_selection.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import { EuiButtonGroup } from '@elastic/eui';
+import { useUrlStorage } from '../hooks/use_url_strorage';
+import { METRIC_TYPE } from '../configurations/constants';
+
+export const MetricSelection = ({ seriesId }: { seriesId: string }) => {
+  const toggleButtons = [
```

Comment:
These buttons seemed a bit smaller compared to the other buttons next to them in your demo. I think they should have the same size.